fs/testdata/copy_file_bom.txt

```diff
@@ -0,0 +1 @@
+﻿hello
```

Comment:
Updated this file a bit. This file now has the below data (in binary):

<img width="685" alt="Screenshot 2025-02-21 at 11 04 30" src="https://github.com/user-attachments/assets/4885d816-2620-43cc-84bd-74042e7ade0d" />

Reply:
Thank you